tests/scripts/run_tests.py

```diff
@@ -490,34 +490,52 @@ def __init__(self, *args, **kwargs):
 
 
 class TestCLI(TestCase):
-    def get_std_args(self, input):
-        return ["zcbor", "--cddl", str(p_manifest12), "--default-max-qty", "16", "convert", "--input", str(input), "-t", "SUIT_Envelope_Tagged"]
+    def get_std_args(self, input, cmd="convert"):
+        return ["zcbor", cmd, "--cddl", str(p_manifest12), "--input", str(input), "-t", "SUIT_Envelope_Tagged"]
 
     def do_testn(self, n):
+        call00 = Popen(self.get_std_args(p_test_vectors12[n], cmd="validate"))
         call0 = Popen(self.get_std_args(p_test_vectors12[n]) + ["--output", "-", "--output-as", "cbor"], stdout=PIPE)
+        call00.communicate()
```

Comment:
Why do we need these extra calls?

Reply:
I split the "convert" command into "validate" and "convert". the new calls test the new "validate" command.